views/authentication/forms/login.hbs

```diff
@@ -76,6 +77,34 @@
 					{{/if}}
 				</div>
 			</form>
+			{{/unless}}
+
+			{{#if (getConfig "FEATURE_NEW_LOGIN_ENABLED")}}
+			<form method="post" action="/login" class="login-form">
```

Comment:
How users might log in with default login (not OAuth) when the flag is enabled? 
The default form should be extended (step 3 in https://ticketsystem.hpi-schul-cloud.org/browse/N21-1) by an additional button that replaces user/password form fields and password reset link when the system has oauth enabled as login type.

Reply:
Which means we might have a system selected, and need to know that 
- oauth is available and allowed and 
- login with username/password has been disabled